contracts/linear/src/lib.rs

```diff
@@ -42,10 +38,7 @@ pub(crate) enum StorageKey {
     Accounts,
     Shares,
     Beneficiaries,
-    Validators, // V0 (Don't comment this)
-    Farms,
-    // AuthorizedUsers,
-    AuthorizedFarmTokens,
+    Validators, // V0 (Don't comment out this)
```

Comment:
We can't remove these storage key, this will affect `Managers` and `ValidatorsV1`